src/libsync/theme.cpp

```diff
@@ -543,25 +543,35 @@ QString Theme::gitSHA1() const
     return devString;
 }
 
-QString Theme::about() const
+QString Theme::aboutInfo() const
 {
     // Shorten Qt's OS name: "macOS Mojave (10.14)" -> "macOS"
     QStringList osStringList = Utility::platformName().split(QLatin1Char(' '));
     QString osName = osStringList.at(0);
 
     QString devString;
     //: Example text: "<p>Nextcloud Desktop Client</p>"   (%1 is the application name)
-    devString = tr("<p>%1 Desktop Client</p>")
-              .arg(APPLICATION_NAME);
+    devString = tr("<p>%1 Desktop Client</p>").arg(APPLICATION_NAME);
 
     devString += tr("<p>Version %1. For more information please click <a href='%2'>here</a>.</p>")
-              .arg(QString::fromLatin1(MIRALL_STRINGIFY(MIRALL_VERSION)) + QString(" (%1)").arg(osName))
-              .arg(helpUrl());
+                     .arg(QString::fromLatin1(MIRALL_STRINGIFY(MIRALL_VERSION)) + QString(" (%1)").arg(osName))
```

Comment:
please use the `arg` overload taking multiple strings at once. they are faster

Reply:
maybe even try to change this long complicated method to build teh string by using placeholders and a single `arg` call ?

Reply:
@mgallien modified where string translations don't break